src/main/java/frc/robot/controls/Driver.java

```diff
@@ -2,20 +2,38 @@
 
 import edu.wpi.first.wpilibj.Joystick;
 import frc.robot.Constants;
+import java.lang.Math;
 
 public class Driver
 {
     public static Joystick throttle = new Joystick(Constants.THROTTLE_JOYSTICK);
     public static Joystick steer = new Joystick(Constants.STEER_JOYSTICK);
 
+    private static double throttleVal = 0;
+    private static double steerVal = 0;
+
     public static double getThrottle()
     {
-        return -throttle.getY();
+        throttleVal = -throttle.getY();
+
+        if (Math.abs(throttleVal) < Constants.DEADZONE)
+        {
+            throttleVal = 0;
+        }
+
+        return throttleVal;
     }
 
     public static double getSteer()
     {
-        return steer.getX();
+        steerVal = steer.getX();
+
+        if (Math.abs(steerVal) < Constants.DEADZONE)
+        {
+            steerVal = 0;
+        }
+
+        return steerVal;
```

Comment:
Same as above

Reply:
Also either call it `steerValue` or just `value` but don't use short forms unnecessarily. The function is called `getSteer()` so it is obvious what the value is, no need to name it `steerVal`